cityofaustin/atd-vz-data
# Migration for blueform location triggers

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13057

This PR converts the SQL for the blueform location trigger in https://github.com/cityofaustin/atd-vz-data/pull/1269 into a migration to apply with the Hasura CLI. Sorry for the double review but using Hasura CLI does make this one faster.

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Use the readme in the `/atd-vzd/` folder to get your local stack up and running with production data and the latest migrations and metadata applied
2. Use the Hasura CLI instructions in that readme to apply the migrations and metadata in this branch. From `/atd-vzd/`:
```bash
hasura migrate apply --envfile .env.local
hasura metadata apply --envfile .env.local
```
3. Follow the test steps 2-4 (the first step of those instructions is to manually make the changes that these migrations do for us) in https://github.com/cityofaustin/atd-vz-data/pull/1269
4. You can also run `./vision-zero hasura-console` to open your local Hasura console and make sure that the [Event Trigger](http://localhost:9695/console/events/data/manage) called `blueform_location_from_latlon` is no longer present in Hasura.
5. If you want to test the down migration, run:
```bash
hasura migrate apply --down 1 --envfile .env.local
```
---
#### Ship list
